fcrepo-clients/src/main/java/edu/unc/lib/dl/acl/fcrepo4/ObjectAclFactory.java

```diff
@@ -200,7 +205,8 @@ public void setRepositoryObjectCacheLoader(RepositoryObjectCacheLoader repositor
             while (it.hasNext()) {
                 Statement stmt = it.next();
                 Property property = stmt.getPredicate();
-                if (CdrAcl.NS.equals(property.getNameSpace())) {
+                if (CdrAcl.NS.equals(property.getNameSpace())
+                        || RDF.type.equals(property) && Cdr.Tombstone.equals(stmt.getResource())) {
```

Comment:
Does this need another set of () around it?

Reply:
hmm, yeah that would be safest. In java `&&` takes precedence over `||` so it'd be evaluated first, which I guess is why this worked, but it'd be safer to group it